### ISSUE TITLE
Menu Buttons

### ISSUE THREAD

**Issue by NovaStellar-DunEDU** [NovaStellar-DunEDU/RockBands#2]
Needs to be created, target size for pointer inputs (or in this case, the buttons) needs to be at least 24 by 24 CSS pixels or larger.

This site fails the minimum for target size.
WCAG-2 checklist, 2.5.8, Navigation issue
